docs/developer/01-implementing-google-analytics.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Analytics
+label: Google Analytics
+---
+
+### Implementation
+
+By adding the following to elements&hellip;
+
+1. `data-ga="click"`
```

Comment:
@armstrongb Probably worth adding the 'error' and 'visible' values you can also add to data-ga, as it's not just click events we can track.

Reply:
Any others that you can think of?

Reply:
No, LGTM